src/polkadot.scss

```diff
@@ -83,6 +83,20 @@ $input-border-color: $gray-300;
 $input--color: $gray-900;
 $input-height: 2.5rem;
 
+.danger, .warning {
```

Comment:
warning should be styled in such a way that it be differentiable fron erros. When it is a warning they technically are able to continue but if it is error the operation will fail or the action is diabled.